package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon --exec babel-node src/index.js",
-    "test": "test"
+    "test": "mocha --require @babel/polyfill --require @babel/register ./src/tests/*.js"
```

Comment:
it could be better if you change the environment before running test 
ex: NODE_ENV=test

Reply:
sounds important!
let me do it.

Reply:
@Niyonsengaeric , 
environment is now set to `test` !